database/group/group.py

```diff
@@ -56,7 +56,7 @@ def get_group(cls, group_id: int) -> "Group":
 				return cls(*cursor.fetchone())
 
 	@classmethod
-	def get_groups(cls) -> Iterable["Group"]:
+	def get_groups(cls) -> Collection["Group"]:
```

Comment:
Could this be instead `list["Group"]`?

Collections seems to be for things like OrderedDict, defaultdict, deque, namedtuple, and user-defined wrappers.

Reply:
Iterable doesn't have len(). So what to use instead?

Reply:
Does `list` work?

Reply:
https://docs.python.org/3/library/collections.abc.html
Collection is anything iterable on which u can use `len` and `in`. examples include list, tuple, set, dict, etc. basically its what we're doing with these objects, so Collection is the right interface to use